smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -1021,6 +1021,104 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
             .$waitAndVerify();
       },
     },
+
+    // Tests that check that requests on an already terminated reader connection
+    // return INVALID_HANDLE.
+    'testTerminatedConnection': {
+      'setUp': async function() {
+        expectReportPlainResult(/*requestId=*/ 101, 'SUCCESS');
+        await mockChromeApi
+            .dispatchEvent(
+                'onDisconnectRequested', /*requestId=*/ 101, readerHandle,
+                chrome.smartCardProviderPrivate.Disposition.LEAVE_CARD)
+            .$waitAndVerify();
+      },
+
+      // Test that Disconnect requested on an already terminated connection
+      // returns INVALID_HANDLE error.
+      'testDisconnect': async function() {
+        expectReportPlainResult(/*requestId=*/ 201, 'INVALID_HANDLE');
+        await mockChromeApi
+            .dispatchEvent(
+                'onDisconnectRequested', /*requestId=*/ 201, readerHandle,
+                chrome.smartCardProviderPrivate.Disposition.LEAVE_CARD)
+            .$waitAndVerify();
+      },
+
+      // Test that Status requested on an already terminated connection
+      // returns INVALID_HANDLE error.
+      'testStatus': async function() {
+        expectReportStatusResult(
+            /*requestId=*/ 201, /*readerName=*/ '',
+            chrome.smartCardProviderPrivate.ConnectionState.ABSENT,
+            chrome.smartCardProviderPrivate.Protocol.UNDEFINED,
+            new ArrayBuffer(0), 'INVALID_HANDLE');
+        await mockChromeApi
+            .dispatchEvent(
+                'onStatusRequested', /*requestId=*/ 201, readerHandle)
+            .$waitAndVerify();
+      },
+
+      // Test that GetAttrib requested on an already terminated connection
+      // returns INVALID_HANDLE error.
+      'testGetAttrib': async function() {
+        expectReportDataResult(
+            /*requestId=*/ 201, new ArrayBuffer(0),
+            /*resultCode=*/ 'INVALID_HANDLE');
+
```

Comment:
nit: Some test cases have this blank line, others don't. Let's make it consistent.

Reply:
done